src/config.json

```diff
@@ -1,5 +1,8 @@
 {
 	"port": 8080,
 	"bodyLimit": "100kb",
-	"corsHeaders": ["Link"]
+	"corsHeaders": ["Link"],
+    "mysql_user": "root",
```

Comment:
Merge in `sectionDAO` and delete this then, either way it can't be merged in to master with this there